source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -54,14 +59,12 @@ def collapse(self, end=False):
 			)
 
 	def move(self, unit, direction, endPoint=None):
-		oldRange = None
+		oldInfo = None
 		if self.basePosition != textInfos.POSITION_CARET:
 			# Insure we haven't gone beyond the visible text.
 			# UIA adds thousands of blank lines to the end of the console.
-			visiRanges = self.obj.UIATextPattern.GetVisibleRanges()
-			visiLength = visiRanges.length
-			if visiLength > 0:
-				oldRange = self._rangeObj.clone()
+			boundingInfo = self.obj.makeTextInfo(textInfos.POSITION_ALL)
```

Comment:
Consistency with other code

```suggestion
			boundingInfo = self.__class__(obj, textInfos.POSITION_ALL)
```